ql/src/test/results/clientnegative/load_data_partition.q.out

```diff
@@ -0,0 +1,15 @@
+PREHOOK: query: drop table if exists validate_load_data
+PREHOOK: type: DROPTABLE
+PREHOOK: Output: database:default
+POSTHOOK: query: drop table if exists validate_load_data
+POSTHOOK: type: DROPTABLE
+POSTHOOK: Output: database:default
+PREHOOK: query: CREATE TABLE validate_load_data(key int, value string) partitioned by (hr int) STORED AS TEXTFILE
+PREHOOK: type: CREATETABLE
+PREHOOK: Output: database:default
+PREHOOK: Output: default@validate_load_data
+POSTHOOK: query: CREATE TABLE validate_load_data(key int, value string) partitioned by (hr int) STORED AS TEXTFILE
+POSTHOOK: type: CREATETABLE
+POSTHOOK: Output: database:default
+POSTHOOK: Output: default@validate_load_data
+#### A masked pattern was here ####
```

Comment:
Here there is no error,  I think it is failing before the expected scenario.

Reply:
After masking the failure scenario is coming like this, before that 
the error is coming as expected 
"FAILED: SemanticException Line 2:17 Invalid path ''/tmp/kv1.txt'': No files matching path hdfs://localhost:65046/tmp/kv1.txt"